xml/deployment_hwrecommend.xml

```diff
@@ -23,6 +23,172 @@
   basis. If several processes are located on the same machine, the CPU, RAM,
   disk and network requirements need to be added up.
  </para>
+
+<sect1 xml:id="network-overview">
+  <title>Network Overview</title>
+  <para>
+    Ceph has several logical networks:
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>
+        A trusted internal network, the back-end network called the
+        the <literal>cluster network</literal>.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        A public client network called <literal>public network</literal>.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Client networks for gateways, these are optional.
+      </para>
+    </listitem>
+  </itemizedlist>
+   <para>
+     The trusted internal network is the back-end network between the OSD
+     nodes for replication and balancing, it needs roughly three times the
+     bandwidth as the public network.
+     The public network is between clients and gateways on the one side to
+     talk to monitors, managers, MDS nodes, OSD nodes. It is also used by
+     monitors, managers, and MDS nodes to talk with OSD nodes.
+   </para>
+   <figure xml:id="network-overview-figure">
+    <title>Network Overview</title>
+    <mediaobject>
+     <imageobject role="html">
+      <imagedata fileref="network-overview-diagram.png" width="70%" format="PNG"/>
+     </imageobject>
+    </mediaobject>
+   </figure>
+
+  <sect2 xml:id="ceph-install-ceph-deploy-network">
+   <title>Network Recommendations</title>
+   <para>
+    For the &ceph; cluster network environment, we recommend two bonded 25 GbE (or faster)
```

Comment:
```suggestion
    For the &ceph; network environment, we recommend two bonded 25 GbE (or faster)
```
cluster otherwise clashes with cluster network further down.